src/scripts/main.js

```diff
@@ -1,3 +1,304 @@
+        const newCellB = cellB.slice(1).replace(/,/g, '.');
+
+        return newCellB - newCellA;
+      }
+
+      switch (true) {
+        case cellA < cellB:
+          return 1;
+
+        case cellA > cellB:
+          return -1;
+
+        default:
+          return 0;
+      }
+    });
+
+    tableBody.innerHTML = '';
+
+    for (const row of sortedArr) {
+      tableBody.append(row);
+    }
+  }
+};
+
+tableHeaderElements.forEach((element, index) => {
+  element.addEventListener('click', () => {
+    handler(index);
+  });
+});
+
+tableBody.addEventListener('click', (e) => {
+  for (const row of tableBodyElements) {
+    row.classList.remove('active');
+  }
+  e.target.closest('tr').classList.add('active');
+});
+
+const form = document.createElement('form');
+
+form.classList.add('new-employee-form');
+body.append(form);
+
+form.insertAdjacentHTML('beforeend', `
+  <label>
+    ${tableHeaderElements[0].innerText}: 
+      <input name="name" type="text" data-qa="name">
+  </label>
+`);
+
+form.insertAdjacentHTML('beforeend', `
+  <label>
+    ${tableHeaderElements[1].innerText}:
+      <input name="position" type="text" data-qa="position">
+  </label>
+`);
+
+form.insertAdjacentHTML('beforeend', `
+  <label>
+    ${tableHeaderElements[2].innerText}:
+      <select data-qa="office" required></select>
+  </label>
+`);
+
+form.insertAdjacentHTML('beforeend', `
+  <label>
+    ${tableHeaderElements[3].innerText}:
+      <input name="age" type="number" data-qa="age">
+  </label>
+`);
+
+form.insertAdjacentHTML('beforeend', `
+  <label>
+    ${tableHeaderElements[4].innerText}:
+      <input name="salary" type="number" data-qa="salary">
+  </label>
+`);
+
+const selector = document.querySelector('select');
+
+selector.dataset.qa = 'office';
+
+const cities = [
+  `Tokyo`,
+  `Singapore`,
+  `London`,
+  `New York`,
+  `Edinburgh`,
+  `San Francisco`,
+];
+
+for (const city of cities) {
+  const element = document.createElement('option');
+
+  element.innerText = city;
+  selector.append(element);
+}
+
+const item = document.createElement('option');
+
+item.innerText = 'Choose:';
+item.disabled = true;
+item.selected = true;
+selector.prepend(item);
+
+const button = document.createElement('button');
+
+button.innerText = 'Save to table';
+button.setAttribute('type', 'submit');
+form.append(button);
+
+const appropriateNotification = function(title, description, type) {
+  const notification = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  h2.innerText = title;
+  h2.classList.add('title');
+  p.innerText = description;
+
+  notification.append(h2);
+  notification.append(p);
+  notification.classList.add('notification');
+  notification.classList.add(type);
+  notification.dataset['qa'] = 'notification';
+  body.append(notification);
+  setTimeout(() => notification.remove(), 3000);
+};
+
+form.addEventListener('submit', (evt) => {
```

Comment:
```suggestion
form.addEventListener('submit', (event) => {
```
Хоч це і не грає критичної ролі, але прийнято називати цей аргумент event або e